### PR TITLE
BN Translations: Updated translations upto 20%

### PR DESCRIPTION
**Pull request by abhik-99**
This PR aims to include the new translations and minor corrections in the previous ones. This bumps the BN translations completion from 12% to 20%.

Ref: Issue #653

### CODE REVIEW

File: po/bn.po [hunk 1]
```diff
@@ -2020,6 +2026,14 @@ msgid ""
...
 "different IDE available called [RustRover](https://www.jetbrains.com/rust/)."
 msgstr ""
+"রাস্ট ইনস্টল করবার পরে আপনার এডিটর বা আই.ডি.ই সজ্জিত করা দরকার। বেশিরভাগ এডিটর "
+"এইজন্যে [rust-analyzer](https://rust-analyzer.github.io/) এর সাথে যোগাযোগ করে "
+"করতে সক্ষম। [rust-analyzer](https://rust-analyzer.github.io/) [VS Code]"
+"(https://code.visualstudio.com/), [Emacs](https://rust-analyzer.github.io/"
+"manual.html#emacs), [Vim/Neovim](https://rust-analyzer.github.io/manual."
+"html#vimneovim) এবং ইত্যাদির জন্যে স্বয়ং-সম্পূর্ণতা এবং সরাসরি সংজ্ঞায় খুজার মতন "
+"কার্যকারিতা প্রদান করে। আরেটা আই.ডি.ই-ও ব্যবহার করা যেতে পারে এখানে যার নাম "
+"[RustRover](https://www.jetbrains.com/rust/)|"
```
Comment (mirwasi) [inline po/bn.po:2036]:
```suggestion
"[RustRover](https://www.jetbrains.com/rust/)।"
```

File: po/bn.po [hunk 2]
```diff
@@ -2088,95 +2105,125 @@ msgid ""
...
+"[সার্বজনীন স্থাপনা](https://doc.rust-lang.org/cargo/commands/cargo-install."
+"html)"
 
 #: src/cargo/rust-ecosystem.md
 msgid ""
 "It is also extensible with sub command plugins as well (such as [cargo "
 "clippy](https://github.com/rust-lang/rust-clippy))."
 msgstr ""
+"[cargo clippy](https://github.com/rust-lang/rust-clippy)-র মতন নানান উপ-আদেশ "
+"প্লাগিন কাজে লাগিয়েও প্রসারিত করা যায়|"
```
Comment (mirwasi) [inline po/bn.po:2219]:
```suggestion
"প্লাগিন কাজে লাগিয়েও প্রসারিত করা যায়।"
```

File: po/bn.po [hunk 3]
```diff
@@ -2225,13 +2278,18 @@ msgid ""
 "The embedded playgrounds cannot execute unit tests. Copy-paste the code and "
 "open it in the real Playground to demonstrate unit tests."
 msgstr ""
+"অনুবিদ্ধ প্লেগ্রাউন্ডটি একক পরীক্ষা চালাতে পারবে না। একক পরীক্ষাগুলি প্রদর্শনের জন্য "
```
Comment (mirwasi) [inline po/bn.po:2281]: Love this usage of "অনুবিদ্ধ"!

**Review by mirwasi — APPROVED** [review]
Thank you very much, this is a substantial contribution! Apart from some typos (which you can fix now or later), I think this is good to go. The most common one is the usage of vertical bar instead of দাঁড়ি (।)

File: po/bn.po [hunk 4]
```diff
@@ -1263,7 +1258,7 @@ msgid ""
 msgstr ""
 "এটি Google এর অ্যান্ড্রয়েড টিম দ্বারা তৈরি একটি চার দিনের রাস্টশিখবার কোর্স যা "
 "রাস্ট-এর সম্পূর্ণ স্পেকট্রাম, মৌলিক সিনট্যাক্স থেকে শুরু করে জেনেরিক এবং ত্রুটি "
-"পরিচালনার মতো উন্নত বিষয় শিখতে সাহায্য করতে পারে ।"
+"পরিচালনার মতো উন্নত বিষয় শিখতে সাহায্য করতে পারে|"
```
[the remark this answers is not in the document]
Comment (abhik-99) [reply po/bn.po:1261]: Yeah, I am not sure on the keyboard shortcut for this one. That's what led to the usage of pipe "|" being confused on my part.

File: po/bn.po (hunk 3 above)
Comment (abhik-99) [reply po/bn.po:2281]: Thanks!

**Comment by mirwasi** [comment]
> > Thank you very much, this is a substantial contribution! Apart from some typos (which you can fix now or later), I think this is good to go. The most common one is the usage of vertical bar instead of দাঁড়ি (।)
> 
> Yup, but which unicode should be used? Unicode 0964 or 09F7? I got a bit confused here and started using pipe "|".

This is a very good point. Unicode's [recommendation](https://unicode.org/faq/bengali.html#10) (please scroll down to the second last question) is to use the Devanagari danda (U+0964).

**Comment by abhik-99** [comment]
> > > Thank you very much, this is a substantial contribution! Apart from some typos (which you can fix now or later), I think this is good to go. The most common one is the usage of vertical bar instead of দাঁড়ি (।)
> > 
> > 
> > Yup, but which unicode should be used? Unicode 0964 or 09F7? I got a bit confused here and started using pipe "|".
> 
> This is a very good point. Unicode's [recommendation](https://unicode.org/faq/bengali.html#10) (please scroll down to the second last question) is to use the Devanagari danda (U+0964).
> 
> Btw, "Bengali Currency Numerator Four" (U+ 09F7) is something else and not to be used as dari.

Thanks! I have incorporated the changes discussed and squashed the commits.

**Pull request merged**